_data/converters.yml

```diff
@@ -0,0 +1,150 @@
+# template
+#     - name:
+#       url: URL to repo
+#       data_types: MRI, EEG, MEG, iEEG, events
+#       expected_input: DICOM, PAR/REC, tsv
+#       language: python, matlab, docker
+#       documentation: URL to documentation
+#       comment:
+#       GUI: false
+
+- group:
```

Comment:
Another group could be physio converters. Some are integrated into general-purpose converters, but still exist and can be used on their own (e.g., `phys2bids`).

Reply:
I don't know if `convert_eprime` counts as a BIDS conversion tool, but happy to see it included!

Reply:
trust me given the "everything but the kitchen sink" nature of the things already in that page, `convert_eprime` belongs there.